src/app/app-routing.module.ts

```diff
@@ -27,6 +27,10 @@ const routes: Routes = [
     path: 'qr-scanner',
     loadChildren: () => import('./qr-scanner/qr-scanner.module').then( m => m.QrScannerPageModule)
   },
+  {
+    path: 'download-list',
```

Comment:
Change the path to "project-downloads" or "downloads"

Reply:
done